packages/ui/src/app/pages/Election/PastElections/PastElections.tsx

```diff
@@ -65,7 +66,7 @@ export const PastElections = () => {
   return <PageLayout header={header} main={displayMain()} />
 }
 
-export const PastElectionsColLayout = '48px 176px 156px 100px 100px'
+export const PastElectionsColLayout = '48px 100px 100px 100px 80px 80px'
```

Comment:
There's enough space even on 1280px screen to keep the column headers on one line:
```suggestion
export const PastElectionsColLayout = '48px 176px 140px 140px 100px 100px'
```
![image](https://github.com/Joystream/pioneer/assets/6571453/e280c105-2694-4b82-ae29-9fdb4d4698d5)